ocaml/testsuite/tests/typing-modal-kinds/basics.ml

```diff
@@ -683,3 +679,130 @@ let foo : (string -> string) -> (string -> string) @ unique
 [%%expect{|
 val foo : (string -> string) -> unique_ (string -> string) = <fun>
 |}]
+
+(* CR layouts: maybe use the helper function below to refactor the tests above,
```

Comment:
I think these new tests are good, but I don't see a need to refactor the old ones. Diversity in test structure is good.

Reply:
Sounds good - should I remove the CR, then?

Reply:
Yes, please. Thanks.